src/main/java/com/n1analytics/paillier/EncryptedNumber.java

```diff
@@ -416,4 +416,28 @@ public boolean equals(EncryptedNumber o) {
             context.equals(o.context) &&
             ciphertext.equals(o.ciphertext));
   }
+
+  /**
+   * Return a byte array representing the underlying raw number.
```

Comment:
Fair enough. FWIW the vagueness was a deliberate part of the trade-off between documenting the function but making it "scary" and discouraging its use.